scripts/genetic_algo/evaluation.py

```diff
@@ -31,7 +31,7 @@ def runzen(cmdline):
 
 
 class Andividual(Individual):
-    BASE_COMMAND = '/app/zenbot.sh sim {instrument} --strategy {strategy} --avg_slippage_pct 0.33 --filename temp.html'
+    BASE_COMMAND = 'env node ../../zenbot.js sim {instrument} --strategy {strategy} --avg_slippage_pct 0.33 --filename temp.html'
```

Comment:
I understand the `../../` but why can't you use `zenbot.sh`?

Reply:
because, otherwise I have this error:


module.js:473
      throw err;
      ^

Error: Cannot find module '/home/emanuele/Code/zenbot.gh/scripts/genetic_algo/zenbot.js'
    at Function.Module._resolveFilename (module.js:527:15)
    at Function.Module._load (module.js:453:25)
    at Function.Module.runMain (module.js:665:10)
    at startup (bootstrap_node.js:201:16)
    at bootstrap_node.js:626:3


it expects to find zenbot.js in current directory that is scripts/genetic_algo/